### PR TITLE
{Branch Management} Merge release 2.34.1 back to dev

### PR DESCRIPTION
**Pull request by zhoxing-ms**
Merge fix #21486 back to `dev`.

🛑 Don't squash merge but merge it locally and push.

### CODE REVIEW

**Comment by yonzhan** [comment]
Branch Management

**Review by evelyn-ys — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Review by jsntcy — APPROVED** [review]

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,21 @@
 Release History
 ===============
 
+2.34.1
+++++++
+
+**Compute**
```
Comment (panchagnula) [inline src/azure-cli/HISTORY.rst:9]: This should be App Service - are we putting this under compute now?

**Review by panchagnula — APPROVED** [review]
LGTM

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:9]:
```suggestion
**App Service**
```

Comment (zhoxing-ms) [reply src/azure-cli/HISTORY.rst:9]: Thank you for your reminder!
I also revised the title in release notes https://github.com/MicrosoftDocs/azure-docs-cli/pull/2998/files#r818235506

**Pull request merged**